vpr/src/device/rr_graph_view.h

```diff
@@ -54,6 +54,16 @@ class RRGraphView {
      * kind of accessors
      */
   public:
+    /** @brief Iterate over nodes. This function is inlined for runtime optimization. */
```

Comment:
Would you mind borrow the comments from rr_graph_obj.h

https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/e6c20492832f1950247c8db7b58274664d190992/vpr/src/device/rr_graph_obj.h#L227-L242

It will help developers to quickly learn how to use.